queries/lmfs/basics/active_tasks.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
```

Comment:
Worth writing examples as .sql files with named parameters?  Then wrapping not-a-test.sh could just call bq directly.  Would remove one layer (nodejs) of indirection.